### PR TITLE
Add a page about how to re-encrypt data source details

### PR DESCRIPTION
**Pull request by susodapop**
## Description

We added this CLI in V10. How about some docs?

### CODE REVIEW

File: src/pages/kb/open-source/admin-guide/secrets.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+The cookie secret is taken from the `REDASH_COOKIE_SECRET` environment variable. It is used for various cryptographic features of the web server, such as authenticating users, signing cookies, and storing user session information. It is required to start the application.
+
+If you attempt to start Redash without it, the webserver will not initialise and you will find the following message in your logs:
+
+```bash
+Exception: You must set the REDASH_COOKIE_SECRET environment variable. Visit http://redash.io/help/open-source/admin-guide/secrets for more information.
+```
+
+To fix this message: create an environment variable called `REDASH_COOKIE_SECRET` and give it a value.
+### Data Source Secret
```
Comment (arikfr) [inline src/pages/kb/open-source/admin-guide/secrets.md:23]: We actually encrypt alert destination options with it too, so probably better to call something else.

File: src/pages/kb/open-source/admin-guide/secrets.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+The data source secret is taken from the `REDASH_SECRET_KEY` environment variable. It is used to encrypt database passwords, API keys, connection strings, and any other secret fields found on the Settings > Data Sources screen.
```
Comment (arikfr) [inline src/pages/kb/open-source/admin-guide/secrets.md:25]: We encrypt the whole configuration object of the data source / alert destination - not only the secrets.

File: src/pages/kb/open-source/admin-guide/secrets.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+It is required to start the application. However, if you do not set one explicitly, Redash will use the cookie secret instead. This is helpful for development. But for maximum security, we recommend you set a unique value for both variables.
+
+
+## Choosing a Secret Key
+
+Treat these secret keys like passwords. A strong secret key will be unique and hard to guess. Do not reuse secret keys across instances of Redash or commit them to version control. If a hostile actor guesses your secret key they might compromise your Redash instance. 
+
+Our cloud images use the CLI tool `pwgen` to generate secret keys, but any [strong password generator](https://duckduckgo.com/?q=pwgen+32+strong) will work:
```
Comment (arikfr) [inline src/pages/kb/open-source/admin-guide/secrets.md:34]: Is this aligned w/ Flask's recommendations?

File: src/pages/kb/open-source/admin-guide/secrets.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+---
+category: admin-guide
+parent_category: open-source
+title: Secrets Keys in Redash
```
Comment (arikfr) [inline src/pages/kb/open-source/admin-guide/secrets.md:4]:
```suggestion
title: Secret Keys in Redash
```

File: src/pages/kb/open-source/admin-guide/secrets.md (hunk 1 above)
Comment (susodapop) [reply src/pages/kb/open-source/admin-guide/secrets.md:23]: Fixed.

File: src/pages/kb/open-source/admin-guide/secrets.md (hunk 2 above)
Comment (susodapop) [reply src/pages/kb/open-source/admin-guide/secrets.md:25]: Fixed.

File: src/pages/kb/open-source/admin-guide/secrets.md (hunk 3 above)
Comment (susodapop) [reply src/pages/kb/open-source/admin-guide/secrets.md:34]: [Here's](https://flask.palletsprojects.com/en/2.0.x/tutorial/deploy/#configure-the-secret-key) what they recommend. I'm updating our guidance to mirror it.

File: src/pages/kb/open-source/admin-guide/secrets.md (hunk 4 above)
Comment (susodapop) [reply src/pages/kb/open-source/admin-guide/secrets.md:4]: Fixed

**Review by arikfr — APPROVED** [review]

**Pull request merged**